vizro-core/tests/unit/vizro/managers/test_model_manager.py

```diff
@@ -0,0 +1,209 @@
+from typing import Any
+
+import pytest
+
+import vizro.models as vm
+from vizro.managers import model_manager
+from vizro.managers._model_manager import FIGURE_MODELS
+
+
+@pytest.fixture
+def page_1(standard_px_chart):
+    return vm.Page(
+        id="page_1_id",
+        title="Page 1",
+        components=[vm.Button(id="page_1_button_id"), vm.Graph(id="page_1_graph_id", figure=standard_px_chart)],
+        controls=[],
+    )
+
+
+@pytest.fixture
+def managers_dashboard_two_pages(vizro_app, page_1, standard_kpi_card):
+    return vm.Dashboard(
+        pages=[
+            page_1,
+            vm.Page(
+                id="page_2_id",
+                title="Page 2",
+                components=[
+                    vm.Button(id="page_2_button_id"),
+                    vm.Figure(id="page_2_figure_id", figure=standard_kpi_card),
+                ],
+            ),
+        ]
+    )
+
+
+pytestmark = pytest.mark.usefixtures("managers_dashboard_two_pages")
```

Comment:
Why not just `autouse=True` the fixture instead? That will do the same. I would only use this alternative notation that you have if the fixture had been defined in a different file without `autouse=True`.